http_request_translator/base.py

```diff
@@ -26,6 +28,9 @@ class AbstractScript(object):
     code_search = ''
     code_nosearch = ''
 
+    _base_module = __name__.split('.', 1)[0]
```

Comment:
Do we need these two attributes to be class attributes? We could have them in `load_attributes`` directly no?